Template.php

```diff
@@ -418,6 +418,7 @@ public function add_if_new($param_name, $value) {
         return FALSE;
         
       case "periodical": case "journal":
+        if (strtolower($this->get('journal')) === 'arxiv e-prints') $this->forget('journal'); // Update to real data
```

Comment:
Should we instead check that the journal title is not any of BAD_TITLES?

Reply:
Excellent point.  Will do.   I will rue the day when a hippie starts the open access data journal “Unknown”